tests/tests/website_footer_tests.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Documentation  Smoke Tests
+Library        OperatingSystem
+Library        SeleniumLibrary
+Library        String
+Resource       ../src/page_objects/keywords/common.robot
+Resource       ../src/page_objects/keywords/home_page.robot
+Resource       ../src/page_objects/keywords/ubuntu.robot
+Resource       ../src/test_files/test_files.resource
+Library        ../src/page_objects/libraries/browser_lib.py
+Suite Setup   Prepare Test Environment
+Test Setup    Prepare For Test
+Test Teardown    Clean After Test
+
+
+*** Test Cases ***
+Find and open About page
+    [documentation]  Click button About and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button About, clicks it and verifies if a list with some text dislays
+    Click Element  ${AboutButton}
+    Wait Until Element Is Visible   ${AboutSubpageText}    1
+
+Find and open Known Issues page
+    [documentation]  Click button Known Issues and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button Known Issues, clicks it and verifies if a list with some text dislays
+    Click Element  ${KnownIssuesButton}
+    Wait Until Element Is Visible  //div[@id="issues"]/ul/*
+    
+Find and open Open source technologies page
+    [documentation]  Click button Open source technologies and check if subpage is opened
+    [tags]  Regression
+    #TODO: Find button Open source technologies, clicks it and verifies if a list with some text dislays
+    Click Element  ${OpenSourceTechnologiesButton}
+    Wait Until Element Is Visible  //div[@id="techs"]/ul/*
+
+Find and open Github repository page
+    [documentation]  Click button Github repository and check if subpage is opened
+    [tags]  Regression
+    Click Element       xpath=/html/body/div[2]/small/a[4]
+    Wait Until Element Is Visible  xpath=/html
```

Comment:
"xpath=/html" will be valid for every html page so it does not check if we are redirected to project's github. I think you could use here something like https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Location%20Should%20Be and check page address. If there would be a need to check other tab you can use https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Switch%20Window